_source/_posts/2022-08-30-introducing-the-new-okta-mobile-sdks.md

```diff
@@ -0,0 +1,333 @@
+
+You can override everything, from the way ID token validation occurs to how tokens are securely stored, or your app can accept the out-of-the-box defaults.
+
+### Support for new Okta features
+
+As Okta continues to innovate, introducing new features and capabilities, our mobile SDKs need to update to take advantage of them. By refactoring these SDKs, we aim to make it as easy as possible for you to adopt new features, which has been our primary motivation. Let's take a look at a few of the more exciting features we now support.
+
+Many platform features center around support for new authentication flows. Classic web-based sign-in, through OIDC, is implemented using the OAuth2 Authorization Code Flow. Instead of locking this functionality away, we've exposed it, along with our new flows. We tried our best to follow similar patterns for all [authentication flows](https://okta.github.io/okta-mobile-swift/development/oktaoauth2/documentation/oktaoauth2/introductiontoauthenticationflows), to reduce a lot of the guesswork necessary to get a feature to work.
+
+## Benefits of the new Okta Mobile SDK
+
+As mentioned above, our aim has been to do more than build parity functionality with the legacy OIDC SDKs. We wanted to unlock amazing new capabilities while simultaneously simplifying your developer experience. So let's take a look at a few of the improvements we've made.
+
+### Simplified web-based authentication
+
+Signing in with a web browser using OIDC should be a straightforward process, and it is the easiest way to get started using Okta in a mobile application. And yet, we felt we could make it even easier.
+
+Just like within Okta OIDC, the [WebAuthenticationUI](https://okta.github.io/okta-mobile-swift/development/webauthenticationui/documentation/webauthenticationui) library is capable of using a configuration file named Okta.plist to define your client settings. From there, you can sign into your application using a single line of code.
+
+```swift
+// Sign in using the default configuration
+let token = try await WebAuthentication.shared?.signIn(from: view.window)
+
+// Save the user's token
+let credential = try Credential.store(token)
+```
+
+The [shared property](https://okta.github.io/okta-mobile-swift/development/webauthenticationui/documentation/webauthenticationui/webauthentication/shared) simplifies how to create a [WebAuthentication](https://okta.github.io/okta-mobile-swift/development/webauthenticationui/documentation/webauthenticationui/webauthentication) instance, but if you want direct control of the configuration, you can simply create one in code.
+
+```swift
+let auth = WebAuthentication(
+    issuer: URL(string: "https://dev-133337.okta.com"),
```

Comment:
@mikenachbaur-okta I had to change this from example.okta.com
`dev-133337` is a domain reserved for the blog, using something like `https://{yourOktaDomain}` would work too, your call 😄

Reply:
I usually use `example.okta.com`, how about that instead?

Reply:
We actually have a lint rule against using `example.okta.com` 😰